### PR TITLE
feature/AT-UN-UI-Unit_Test_Admin_Catch

### PR DESCRIPTION
**Pull request by acones96**

### CODE REVIEW

File: src/controllers/admin.controller.js [hunk 1]
```diff
@@ -70,18 +61,14 @@ adminCtrl.renderEditCourseForm = async (req, res) => {
         let responseCourses = await universityServiceAPI.getAllCourses();
         courseDetails = responseCourses.data.filter(function (c) { return c._id == courseId; });
 
-        if (courseDetails.length == 0){
```
Comment (at-internship) [inline src/controllers/admin.controller.js:73]: Why did you remove this validation?

**Review by at-internship — CHANGES_REQUESTED** [review]

File: src/controllers/admin.controller.js (hunk 1 above)
Comment (acones96) [reply src/controllers/admin.controller.js:73]: I updated again, I was trying something with it but forgot to put it again

**Review by at-internship — APPROVED** [review]
Approved

**Review by raaul-at — APPROVED** [review]
